### PR TITLE
Rename rewind to restore

### PR DESCRIPTION
**Pull request by enejb**
## Changes proposed in this Pull Request
Update the wording from rewind to restore. 

## Before
Screenshot 2019-12-02 at 16 10 37
Screenshot 2019-12-02 at 16 10 52
Screenshot 2019-12-02 at 16 11 31
Screenshot 2019-12-02 at 16 17 57

## After

Screenshot 2019-12-02 at 16 10 42

Screenshot 2019-12-02 at 16 11 00
Screenshot 2019-12-02 at 16 11 10
Screenshot 2019-12-02 at 16 17 51

## Testing instructions
* Go to Tools -> Activity for a site using Jetpack Backup, enter site server credentials if necessary, and start a restore. Check all the copy.

### CODE REVIEW

File: client/blocks/disconnect-jetpack/index.jsx [hunk 1]
```diff
@@ -281,10 +281,10 @@ class DisconnectJetpack extends PureComponent {
 					className="disconnect-jetpack__try-rewind disconnect-jetpack__block"
 				>
 					<p className="disconnect-jetpack__highlight">
-						{ translate( 'Experiencing connection issues? Try to go back and rewind your site.' ) }
+						{ translate( 'Experiencing connection issues? Try to go back and restore your site.' ) }
```
Comment (a8ci18n) [inline client/blocks/disconnect-jetpack/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/blocks/disconnect-jetpack/index.jsx [hunk 2]
```diff
@@ -281,10 +281,10 @@ class DisconnectJetpack extends PureComponent {
...
 					</p>
 					<div className="disconnect-jetpack__try-rewind-button-wrap">
-						<Button onClick={ this.handleTryRewind }>{ translate( 'Rewind site' ) }</Button>
+						<Button onClick={ this.handleTryRewind }>{ translate( 'Restore site' ) }</Button>
```
Comment (a8ci18n) [inline client/blocks/disconnect-jetpack/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Restore Site' )` *ES Score: 10*

File: client/my-sites/activity/activity-log-banner/progress-banner.jsx [hunk 3]
```diff
@@ -67,14 +67,14 @@ function ProgressBanner( {
 						? translate( 'The cloning process will start in a moment.' )
 						: translate( 'Away we go! Your site is being cloned.' );
 			} else {
-				title = translate( 'Currently rewinding your site' );
+				title = translate( 'Currently restoring your site' );
 				description = translate(
-					"We're rewinding your site back to %(dateTime)s. You'll be notified once it's complete.",
+					"We're restoring your site back to %(dateTime)s. You'll be notified once it's complete.",
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-banner/progress-banner.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'We're in the process of restoring your site back to %(dateTime)s. You'll be notified once it's complete.' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/rewind-with-restore)

File: client/my-sites/activity/activity-log-item/index.jsx [hunk 4]
```diff
@@ -338,20 +338,20 @@ class ActivityLogItem extends Component {
 				{ mightRewind && (
 					<ActivityLogConfirmDialog
 						key="activity-rewind-dialog"
-						confirmTitle={ translate( 'Confirm Rewind' ) }
+						confirmTitle={ translate( 'Confirm Restore' ) }
 						notice={
 							this.state.disableRestoreButton
-								? translate( 'Please select at least one item to rewind.' )
+								? translate( 'Please select at least one item to restore.' )
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-item/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/activity/activity-log-banner/progress-banner.jsx [hunk 5]
```diff
@@ -67,15 +67,15 @@ function ProgressBanner( {
...
 					{ args: { dateTime } }
 				);
 				statusMessage =
 					'queued' === status
-						? translate( 'Your rewind will start in a moment.' )
-						: translate( 'Away we go! Your site is being rewound.' );
+						? translate( 'Your restore will start in a moment.' )
+						: translate( 'Away we go! Your site is being restored.' );
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-banner/progress-banner.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/activity/activity-log-item/index.jsx [hunk 6]
```diff
@@ -338,20 +338,20 @@ class ActivityLogItem extends Component {
...
 						onClose={ this.cancelRewindIntent }
 						onConfirm={ this.confirmRewind }
 						onSettingsChange={ this.restoreSettingsChange }
 						supportLink="https://jetpack.com/support/how-to-rewind"
-						title={ translate( 'Rewind Site' ) }
+						title={ translate( 'Restore Site' ) }
 						disableButton={ this.state.disableRestoreButton }
 					>
-						{ translate( '{{time/}} is the selected point for your site Rewind.', {
+						{ translate( '{{time/}} is the selected point for your site Restore.', {
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-item/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by seear** [comment]
Rebased to master.

File: client/my-sites/activity/activity-log-confirm-dialog/index.jsx [hunk 7]
```diff
@@ -49,7 +49,7 @@ const ActivityLogConfirmDialog = ( {
 			<div className="activity-log-confirm-dialog__partial-restore-settings">
 				<p>
 					{ notice
-						? translate( 'Choose the items you wish to rewind:' )
+						? translate( 'Choose the items you wish to restore:' )
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-confirm-dialog/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/activity/activity-log/rewind-unavailability-notice.jsx [hunk 8]
```diff
@@ -33,7 +33,7 @@ export const RewindUnavailabilityNotice = ( {
 					href={ adminUrl }
 					title={ translate( 'The site is not connected.' ) }
 					description={ translate(
-						"We can't back up or rewind your site until it has been reconnected."
+						"We can't back up or restore your site until it has been reconnected."
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log/rewind-unavailability-notice.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/activity/activity-log-item/index.jsx [hunk 9]
```diff
@@ -336,20 +336,20 @@ class ActivityLogItem extends Component {
...
 						onClose={ this.cancelRewindIntent }
 						onConfirm={ this.confirmRewind }
 						onSettingsChange={ this.restoreSettingsChange }
 						supportLink="https://jetpack.com/support/how-to-rewind"
-						title={ translate( 'Rewind Site' ) }
+						title={ translate( 'Restore Site' ) }
 						disableButton={ this.state.disableRestoreButton }
 					>
-						{ translate( '{{time/}} is the selected point for your site Rewind.', {
+						{ translate( '{{time/}} is the selected point for your site restore.', {
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-item/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Review by seear — APPROVED** [review]
:shipit:

**Pull request merged**